src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -433,5 +433,14 @@ private static void NewMethod(out Construct obj1, out Construct obj2, out Constr
 
 compareTokens: false);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractMethod)]
+        public async Task TestTuple()
+        {
+            await TestAsync(
+@"class Program { static void Main ( string [ ] args ) { [| (int, int) x = (1, 2); |]  System . Console . WriteLine ( x.Item1 ); } } ",
```

Comment:
We have a tool that generates these from within EditorTestApp.  This is how the tool formats things.

Note: you don't have tomimick the other tests.  You can write these tests over multiple lines, with whatever spacing you prefer if you want.